test/cli/parse.spec.ts

```diff
@@ -115,10 +115,13 @@ describe("command line", () => {
             ["luaTarget", "5.2", { luaTarget: tstl.LuaTarget.Lua52 }],
             ["luaTarget", "5.3", { luaTarget: tstl.LuaTarget.Lua53 }],
             ["luaTarget", "jit", { luaTarget: tstl.LuaTarget.LuaJIT }],
-        ])("--%s %s", (optionName, value, expected) => {
+
+            ["luaBundle", "foo", { luaBundle: "foo" }],
+            ["luaBundleEntry", "bar", { luaBundleEntry: "bar" }],
+        ])("{ %p: %p }", (optionName, value, expected) => {
```

Comment:
In that case I think it looks better because of similarity to actual command line
```suggestion
        ])("--%s %s", (optionName, value, expected) => {
```